src/test/java/executor/service/service/ParalleFlowExecutorServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package executor.service.service;
+
+import executor.service.model.Scenario;
+import executor.service.model.Step;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+public class ParalleFlowExecutorServiceTest {
+
+    private ScenarioExecutor scenarioExecutor;
+    private WebDriver webDriver;
+    private ParalleFlowExecutorService parallelFlow;
+
+    @BeforeEach
+    void setUp() {
+        this.scenarioExecutor = mock(ScenarioExecutor.class);
+        this.webDriver = mock(WebDriver.class);
+        this.parallelFlow = new ParalleFlowExecutorService(scenarioExecutor);
+    }
+
+    @AfterEach
+    void tearDown() {
+        parallelFlow.shutdown();
+    }
+
+    @Test
+    public void test() {
+        List<Step> steps1 = new ArrayList<>();
+        steps1.add(new Step("clickCss", "body > ul > li:nth-child(1) > a"));
+        steps1.add(new Step("sleep", "5:10"));
+        steps1.add(new Step("clickXpath", "/html/body/p"));
```

Comment:
code duplication

Reply:
these are different steps from the technical task